earth_enterprise/src/installer/install_fusion.sh

```diff
@@ -68,6 +68,7 @@ ROOT_USERNAME="root"
 SUPPORTED_OS_LIST=("Ubuntu", "Red Hat Enterprise Linux (RHEL)")
 UBUNTUKEY="ubuntu"
 REDHATKEY="rhel"
+CENTOSKEY="centos"
```

Comment:
I apparently can't comment on the relevant line, but on line 268, "Mame" should be "Name".  Might as well fix that here.